scripts/missions/amk/09_Rescue_A_Moogles_Labor_of_Love.lua

```diff
@@ -14,83 +27,202 @@ mission.reward =
     nextMission = { xi.mission.log_id.AMK, xi.mission.id.amk.ROAR_A_CAT_BURGLAR_BARES_HER_FANGS },
 }
 
+local markerIds =
+{
+    -- [qm#] = marker ID
+    [1]  = 17629668,
+    [8]  = 17629752,
+    [9]  = 17629753,
+    [10] = 17629754,
+    [11] = 17629755,
+    [12] = 17629756,
+    [13] = 17629757,
+    [14] = 17629758,
+    [15] = 17629759,
+    [16] = 17629760,
+    [17] = 17629761,
+    [18] = 17629762,
+    [19] = 17629763,
+    [20] = 17629764,
+    [21] = 17629765,
+    [22] = 17629766,
+    [23] = 17629767,
+    [24] = 17629768,
+    [26] = 17629770,
+    [27] = 17629771,
+}
```

Comment:
Using raw IDs for NPCs or mobs in not "zone" files is not a desirable method.
The main reason is because this IDs can and will shift and theres no way for us to remember this IDs have to be updated.

I would recommend defining this npcs in their zone ID file and call their enums here